src/cascadia/TerminalSettingsModel/CascadiaSettingsSerialization.cpp

```diff
@@ -293,7 +293,17 @@ void SettingsLoader::FinalizeLayering()
     for (const auto& profile : userSettings.profiles)
     {
         profile->AddMostImportantParent(userSettings.baseLayerProfile);
+
+        // This completes the parenting process that was started in _addUserProfileParent().
         profile->_FinalizeInheritance();
+        if (profile->Origin() == OriginTag::None)

```

Comment:
Which profiles would have an origin of "None"? Can you explain in a comment?

Reply:
I see, it's sorta mentioned later...

Reply:
Do you feel like the comment above this is sufficient?